### PR TITLE
Updated Config for Edit UI

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity###

*Description of changes:*
* Updated uses of `Correct-ui` to `Edit-ui`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: public/config/configuration.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
   "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
   "CREATE_URL": "https://business-create-dev.pathfinder.gov.bc.ca/businesses/create/",
-  "CORRECT_URL": "https://business-correct-dev.pathfinder.gov.bc.ca/businesses/correct/",
+  "EDIT_URL": "https://business-correct-dev.pathfinder.gov.bc.ca/businesses/edit/",
```
Comment (severinbeauvais) [inline public/config/configuration.json:5]: Need to change host to `business-create...`.

File: src/components/Dashboard/FilingHistoryList.vue [hunk 2]
```diff
@@ -876,8 +876,8 @@ export default {
           break
         case FilingTypes.INCORPORATION_APPLICATION:
           // redirect to Correct web app to correct this Incorporation Application
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue:878]: Please update comment too :)

**Comment by severinbeauvais** [comment]
Before you merge this, please update the Filings UI Dev config map. The build/deploy will read in the new config map.

File: public/config/configuration.json (hunk 1 above)
Comment (cameron-freshworks) [reply public/config/configuration.json:5]: ah, right! Tho i think the routing might actually business-edit here. Just starting into OS stuff now, i've gotten the run down on what needs to be done 😅

File: src/components/Dashboard/FilingHistoryList.vue (hunk 2 above)
Comment (cameron-freshworks) [reply src/components/Dashboard/FilingHistoryList.vue:878]: good catch, thanks!

File: public/config/configuration.json (hunk 1 above)
Comment (cameron-freshworks) [reply public/config/configuration.json:5]: Fixed to -create in next commit. Can revisit this, if the base route can actually change under the same os project.

**Pull request merged**